mitodl/mit-open
# Use newer Learning Resource list cards in Learning Paths lists

### What are the relevant tickets?

Closes: https://github.com/mitodl/hq/issues/4802

### Description (What does it do?)


Updates the Learning Path details pages to use the newer Learning Resource List Cards in the Items Listing.

### Screenshots (if appropriate):


<img width="928" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/c11b0642-73f0-4d6c-8e39-3530dee5f2a3">


### How can this be tested?


- Sign in
- Navigate to a Learning Path details page, e.g. `/learningpaths/1`
- These should display the LearningResourceListCard component (https://mitodl.github.io/mit-open/?path=/story/smoot-design-cards-learningresourcelistcard--learning-path)
- The Dashboard "My Lists" details pages should still use the condensed version (https://mitodl.github.io/mit-open/?path=/story/smoot-design-cards-learningresourcelistcardcondensed--free-course)

### Additional Context



This change refactors the `ItemsListing` out of `pages/ListDetailsPage` and over to its new `page-component`. The `ListDetailsComponent `(which provides the title, description, sort buttons, etc) is renamed `ItemListingComponent` and moved adjacent. Both were being imported by another page (`DashboardPage`) and pages should not import from each each other.

The `ItemListingComponent` takes a new `condensed` prop, which determines whether to use `LearningResourceListCard` or `LearningResourceListCardCondensed` from `ol-components`.

The items lists remain sortable, though the sort view has not been update to use the newer card or styles.
